libraries/currencyUtils/currency.js

```diff
@@ -29,3 +29,7 @@ export function currencyNormalizer(toCurrency = null, bestEffort = true, convert
 export function currencyCompare(get = (obj) => [obj.cpm, obj.currency], normalize = currencyNormalizer()) {
   return keyCompare(obj => normalize.apply(null, get(obj)))
 }
+
+export function getCurrencyFromBidderRequest(bidderRequest) {
+  return (bidderRequest?.ortb2?.cur || [])[0];
```

Comment:
I think this should be its own library, or even in core. the issue with putting it here is that almost none of the consumers need the rest of this library, so to get a 1 line utility you pull in Kbs of code.